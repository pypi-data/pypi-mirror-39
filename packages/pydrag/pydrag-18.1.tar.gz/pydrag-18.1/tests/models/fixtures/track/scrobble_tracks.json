{
    "interactions": [
        {
            "request": {
                "body": "method=auth.getMobileSession&format=json&api_key=LAST_FM_API_KEY&username=Zaratoustre&authToken=USER_AUTH_TOKEN&api_sig=API_SIG",
                "headers": {
                    "Accept": [
                        "*/*"
                    ],
                    "Accept-Encoding": [
                        "gzip, deflate"
                    ],
                    "Connection": [
                        "keep-alive"
                    ],
                    "Content-Length": [
                        "186"
                    ],
                    "Content-Type": [
                        "application/x-www-form-urlencoded"
                    ],
                    "User-Agent": [
                        "python-requests/2.20.1"
                    ]
                },
                "method": "POST",
                "uri": "https://ws.audioscrobbler.com/2.0/"
            },
            "response": {
                "body": {
                    "string": "{\"session\":{\"subscriber\":0,\"name\":\"Zaratoustre\",\"key\":\"CENSORED\"}}"
                },
                "headers": {
                    "Access-Control-Allow-Methods": [
                        "POST, GET, OPTIONS"
                    ],
                    "Access-Control-Allow-Origin": [
                        "*"
                    ],
                    "Access-Control-Max-Age": [
                        "86400"
                    ],
                    "Connection": [
                        "keep-alive"
                    ],
                    "Content-Type": [
                        "application/json"
                    ],
                    "Date": [
                        "Sat, 08 Dec 2018 21:25:25 GMT"
                    ],
                    "Server": [
                        "openresty/1.9.7.5"
                    ],
                    "content-length": [
                        "90"
                    ]
                },
                "status": {
                    "code": 200,
                    "message": "OK"
                }
            }
        },
        {
            "request": {
                "body": "method=track.scrobble&artist%5B0%5D=Green+Day&track%5B0%5D=Bang+Bang&timestamp%5B0%5D=1541878500&artist%5B1%5D=Awolnation&track%5B1%5D=Sail&timestamp%5B1%5D=1541878500&format=json&api_key=LAST_FM_API_KEY&sk=USER_SESSION&api_sig=API_SIG",
                "headers": {
                    "Accept": [
                        "*/*"
                    ],
                    "Accept-Encoding": [
                        "gzip, deflate"
                    ],
                    "Connection": [
                        "keep-alive"
                    ],
                    "Content-Length": [
                        "297"
                    ],
                    "Content-Type": [
                        "application/x-www-form-urlencoded"
                    ],
                    "User-Agent": [
                        "python-requests/2.20.1"
                    ]
                },
                "method": "POST",
                "uri": "https://ws.audioscrobbler.com/2.0/"
            },
            "response": {
                "body": {
                    "string": "{\"scrobbles\":{\"@attr\":{\"accepted\":0,\"ignored\":2},\"scrobble\":[{\"artist\":{\"corrected\":\"0\",\"#text\":\"Green Day\"},\"ignoredMessage\":{\"code\":\"1\",\"#text\":\"\"},\"albumArtist\":{\"corrected\":\"0\",\"#text\":\"\"},\"timestamp\":\"1541878500\",\"album\":{\"corrected\":\"0\"},\"track\":{\"corrected\":\"0\",\"#text\":\"Bang Bang\"}},{\"artist\":{\"corrected\":\"0\",\"#text\":\"Awolnation\"},\"ignoredMessage\":{\"code\":\"1\",\"#text\":\"\"},\"albumArtist\":{\"corrected\":\"0\",\"#text\":\"\"},\"timestamp\":\"1541878500\",\"album\":{\"corrected\":\"0\"},\"track\":{\"corrected\":\"0\",\"#text\":\"Sail\"}}]}}"
                },
                "headers": {
                    "Access-Control-Allow-Methods": [
                        "POST, GET, OPTIONS"
                    ],
                    "Access-Control-Allow-Origin": [
                        "*"
                    ],
                    "Access-Control-Max-Age": [
                        "86400"
                    ],
                    "Connection": [
                        "keep-alive"
                    ],
                    "Content-Type": [
                        "application/json"
                    ],
                    "Date": [
                        "Sat, 08 Dec 2018 21:26:00 GMT"
                    ],
                    "Server": [
                        "openresty/1.9.7.5"
                    ],
                    "content-length": [
                        "519"
                    ]
                },
                "status": {
                    "code": 200,
                    "message": "OK"
                }
            }
        },
        {
            "request": {
                "body": "method=track.scrobble&artist%5B0%5D=The+Head+and+the+Heart&track%5B0%5D=All+We+Ever+Knew&timestamp%5B0%5D=1541878500&artist%5B1%5D=Kaleo&track%5B1%5D=Way+Down+We+Go&timestamp%5B1%5D=1541878500&format=json&api_key=LAST_FM_API_KEY&sk=USER_SESSION&api_sig=API_SIG",
                "headers": {
                    "Accept": [
                        "*/*"
                    ],
                    "Accept-Encoding": [
                        "gzip, deflate"
                    ],
                    "Connection": [
                        "keep-alive"
                    ],
                    "Content-Length": [
                        "322"
                    ],
                    "Content-Type": [
                        "application/x-www-form-urlencoded"
                    ],
                    "User-Agent": [
                        "python-requests/2.20.1"
                    ]
                },
                "method": "POST",
                "uri": "https://ws.audioscrobbler.com/2.0/"
            },
            "response": {
                "body": {
                    "string": "{\"scrobbles\":{\"@attr\":{\"accepted\":0,\"ignored\":2},\"scrobble\":[{\"artist\":{\"corrected\":\"0\",\"#text\":\"The Head and the Heart\"},\"ignoredMessage\":{\"code\":\"1\",\"#text\":\"\"},\"albumArtist\":{\"corrected\":\"0\",\"#text\":\"\"},\"timestamp\":\"1541878500\",\"album\":{\"corrected\":\"0\"},\"track\":{\"corrected\":\"0\",\"#text\":\"All We Ever Knew\"}},{\"artist\":{\"corrected\":\"0\",\"#text\":\"Kaleo\"},\"ignoredMessage\":{\"code\":\"1\",\"#text\":\"\"},\"albumArtist\":{\"corrected\":\"0\",\"#text\":\"\"},\"timestamp\":\"1541878500\",\"album\":{\"corrected\":\"0\"},\"track\":{\"corrected\":\"0\",\"#text\":\"Way Down We Go\"}}]}}"
                },
                "headers": {
                    "Access-Control-Allow-Methods": [
                        "POST, GET, OPTIONS"
                    ],
                    "Access-Control-Allow-Origin": [
                        "*"
                    ],
                    "Access-Control-Max-Age": [
                        "86400"
                    ],
                    "Connection": [
                        "keep-alive"
                    ],
                    "Content-Type": [
                        "application/json"
                    ],
                    "Date": [
                        "Sat, 08 Dec 2018 21:26:13 GMT"
                    ],
                    "Server": [
                        "openresty/1.9.7.5"
                    ],
                    "content-length": [
                        "544"
                    ]
                },
                "status": {
                    "code": 200,
                    "message": "OK"
                }
            }
        },
        {
            "request": {
                "body": "method=track.scrobble&artist%5B0%5D=Disturbed&track%5B0%5D=The+Sound+of+Silence&timestamp%5B0%5D=1541878500&format=json&api_key=LAST_FM_API_KEY&sk=USER_SESSION&api_sig=API_SIG",
                "headers": {
                    "Accept": [
                        "*/*"
                    ],
                    "Accept-Encoding": [
                        "gzip, deflate"
                    ],
                    "Connection": [
                        "keep-alive"
                    ],
                    "Content-Length": [
                        "237"
                    ],
                    "Content-Type": [
                        "application/x-www-form-urlencoded"
                    ],
                    "User-Agent": [
                        "python-requests/2.20.1"
                    ]
                },
                "method": "POST",
                "uri": "https://ws.audioscrobbler.com/2.0/"
            },
            "response": {
                "body": {
                    "string": "{\"scrobbles\":{\"@attr\":{\"accepted\":0,\"ignored\":1},\"scrobble\":{\"artist\":{\"corrected\":\"0\",\"#text\":\"Disturbed\"},\"ignoredMessage\":{\"code\":\"1\",\"#text\":\"\"},\"albumArtist\":{\"corrected\":\"0\",\"#text\":\"\"},\"timestamp\":\"1541878500\",\"album\":{\"corrected\":\"0\"},\"track\":{\"corrected\":\"0\",\"#text\":\"The Sound of Silence\"}}}}"
                },
                "headers": {
                    "Access-Control-Allow-Methods": [
                        "POST, GET, OPTIONS"
                    ],
                    "Access-Control-Allow-Origin": [
                        "*"
                    ],
                    "Access-Control-Max-Age": [
                        "86400"
                    ],
                    "Connection": [
                        "keep-alive"
                    ],
                    "Content-Type": [
                        "application/json"
                    ],
                    "Date": [
                        "Sat, 08 Dec 2018 21:26:46 GMT"
                    ],
                    "Server": [
                        "openresty/1.9.7.5"
                    ],
                    "content-length": [
                        "302"
                    ]
                },
                "status": {
                    "code": 200,
                    "message": "OK"
                }
            }
        }
    ],
    "version": 1
}
