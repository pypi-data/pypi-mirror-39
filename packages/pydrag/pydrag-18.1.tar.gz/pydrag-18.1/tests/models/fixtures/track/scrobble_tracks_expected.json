{
    "data": [
        {
            "artist": "Green Day",
            "timestamp": 1541878500,
            "track": "Bang Bang"
        },
        {
            "artist": "Awolnation",
            "timestamp": 1541878500,
            "track": "Sail"
        },
        {
            "artist": "The Head and the Heart",
            "timestamp": 1541878500,
            "track": "All We Ever Knew"
        },
        {
            "artist": "Kaleo",
            "timestamp": 1541878500,
            "track": "Way Down We Go"
        },
        {
            "artist": "Disturbed",
            "timestamp": 1541878500,
            "track": "The Sound of Silence"
        }
    ]
}
